int32 team
int32 action
int32 planer_state
float32[] action_pos
int32[] cup
int32[] hand
int32[] reef
bool NS
bool emerg
